pkgs/top-level/all-packages.nix

```diff
@@ -1376,7 +1378,7 @@ in
 
   copyright-update = callPackage ../tools/text/copyright-update { };
 
-  corebird = callPackage ../applications/networking/corebird { };
+  corebird = throw "corebird has been discontinued, use cawbird instead";
```

Comment:
This should be in `aliases.nix` see https://github.com/NixOS/nixpkgs/commit/294093669f9ae5e3e4d79ab2bde673bd10b38bb6

Can you format the message like?
```
throw "deprecated 2019-09-19: See https://www.patreon.com/posts/corebirds-future-18921328. Please use Cawbird as replacement.";
```

etc. or similar